ozaria/site/styles/play/level/tome/spell-palette-view.sass

```diff
@@ -0,0 +1,249 @@
+@import "app/styles/mixins"
```

Comment:
Moved this file from legacy folder to ozaria folder, and involves some minor changes. Code review not necessary.